pages/help.py

```diff
@@ -489,10 +488,9 @@ def get_content(self):
 aware of and are working on.
 """)),
                     FaqItem("kaspersky", _("I have Kaspersky anti-virus, and Freenet doesn't install, or shows \"Download/upload queue database corrupted!\""), _("""
-                    Kaspersky can be a problem with Freenet. See [here](
-https://wiki.freenetproject.org/Installing/Windows#.27Download
-.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7
-.29). We recommend you turn off Kaspersky during install and during node
+                    Kaspersky can be a problem with Freenet.
+See [here](https://wiki.freenetproject.org/Installing/Windows#.27Download.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7.29).
```

Comment:
Ah, being unmaintained could make it a no-go. Oh well.

I'm not intending to suggest changing all the links in the file. Indeed if they aren't currently broken I'd rather they weren't to avoid making more work for translators.